src/active_query.rs

```diff
@@ -53,6 +55,10 @@ pub(crate) struct ActiveQuery {
     /// Stores the values accumulated to the given ingredient.
     /// The type of accumulated value is erased but known to the ingredient.
     pub(crate) accumulated: AccumulatedMap,
+
+    /// [`InputAccumulatedValues::Empty`] if any input read during the query's execution
+    /// has any direct or indirect accumulated values.
+    pub(super) accumulated_inputs: InputAccumulatedValues,
```

Comment:
I think the documentation is now outdated because you changed it to track whether the query itself or any of its dependencies have any accumulated values.